### ISSUE TITLE
importas: config seems to get sorted before execution

### ISSUE THREAD

**Issue by grosser** [golangci/golangci-lint-action#1110]
### Welcome

- [X] Yes, I understand that the GitHub action repository is not the repository of golangci-lint itself.
- [X] Yes, I've searched similar issues on GitHub and didn't find any.
- [X] Yes, I've included all information below (version, config, etc).

### Description of the problem

I'm using `import amissionv1 "k8s.io/api/admissionregistration/v1/admissionregistration/v1"`
and locally it does not complain, but when running CI it does
so I looks like the config is getting sorted/shuffled before execution somehow

### Version of golangci-lint

v1.60.3

### Version of the GitHub Action

4

### Workflow file

<details>

```yml
  lint:
    name: make lint
    runs-on: [self-hosted, zendesk-stable]
    steps:
      - name: Checkout source
        uses: actions/checkout@v4
      - name: Setup Golang
        uses: actions/setup-go@v5
      - name: golangci-lint
        uses: golangci/golangci-lint-action@v4
        with:
          version: v1.60.3
          skip-go-installation: true
          working-directory: workload-migration
```

</details>

### Golangci-lint configuration

<details>

```yml
linters:
  disable-all: true
  enable:
    - importas
linters-settings:
  importas:
    alias:
      - pkg: k8s.io/api/admissionregistration/v1
        alias: admissionv1
      - pkg: k8s.io/api/(\w+)/(v\d+\w*)
        alias: "$1$2"
```

</details>

### Go version

1.22.2

### Code example or link to a public repository

<details>

```go
import amissionv1 "k8s.io/api/admissionregistration/v1/admissionregistration/v1"
```

</details>

**Comment by ldez** [comment]
Hello,

the importas configuration is a slice, so there is no sort.

But, the configuration is set to the linter through the flags, and those flags are stored inside a map.

https://github.com/julz/importas/blob/b8654f969a5f5c63b119cadb1d310322a9d08d00/flags.go#L14

https://github.com/julz/importas/blob/b8654f969a5f5c63b119cadb1d310322a9d08d00/flags.go#L20

So the problem is not inside golangci-lint but inside [importas](https://github.com/julz/importas).

Note: please use the official issue template -> https://github.com/golangci/golangci-lint/issues/new/choose

**Comment by grosser** [comment]
thx. made a fix PR :)
